tests/unit/test-extension-runners/test.extension-runners.js

```diff
@@ -368,6 +405,24 @@ describe('util/extension-runners', () => {
       );
     });
 
+    it('lints changed files', () => {
```

Comment:
@saintsebastian oh yeah, that is definitely missing, we should have some tests similar to the one that inject a fake FirefoxDesktopExtensionRunner into the cmd run and checks that it has been created with the expected parameters.

Thanks for spotting the missing test!